epam-assets/theme/theme_loveship.scss

```diff
@@ -246,69 +258,60 @@
 
     /* not accepted part */
 
-    --uui-disabled: var(--night300); // TODO: name does not clearly reflect the meaning
+    --uui-disabled: var(--neutral30); // TODO: name does not clearly reflect the meaning
 
     /* control specific */
 
     --uui-link: var(--sky);
     --uui-link-hover: var(--sky-dark);
-    --uui-link-wite-hover: var(--night400);
-    --uui-link-black-hover: var(--night800);
+    --uui-link-wite-hover: var(--neutral40);
+    --uui-link-black-hover: var(--neutral80);
     --uui-link-visited: #8018A0;
-    --uui-link-disabled: var(--night500);
+    --uui-link-disabled: var(--neutral50);
 
-    --uui-icon: var(--night600);
-    --uui-icon-hover: var(--night700);
-    --uui-icon-active: var(--night800);
-    --uui-icon-disabled: var(--night500);
+    --uui-icon: var(--neutral60);
+    --uui-icon-hover: var(--neutral70);
+    --uui-icon-active: var(--neutral80);
+    --uui-icon-disabled: var(--neutral50);
 
-    --uui-scroll_bars-bg: var(--night400);
-    --uui-scroll_bars-bg-active: var(--night600);
+    --uui-scroll_bars-bg: var(--neutral40);
+    --uui-scroll_bars-bg-active: var(--neutral60);
 
     --uui-control-bg: var(--uui-surface);
-    --uui-control-bg-hover: var(--night600);
-    --uui-control-bg-disabled: var(--night50);
-    --uui-control-bg-readonly: var(--night50);
-    //--uui-control-bg-checked: var(--uui-primary);
-    //--uui-control-bg-checked-hover: var(--uui-primary-dark);
-    //--uui-control-bg-checked-disabled: var(--uui-primary-light);
-    --uui-control_switch-bg: var(--night400);
+    --uui-control-bg-hover: var(--neutral60);
+    --uui-control-bg-disabled: var(--neutral5);
+    --uui-control-bg-readonly: var(--neutral5);
+    --uui-control_switch-bg: var(--neutral40);
```

Comment:
probably we can rename '--uui-control_switch-bg' to '--uui-switch-bg', because it's already component specific variable. The same for '--uui-control_switch-toggler-bg' and '--uui-control_switch-border-hover'